tests/test_schema.py

```diff
@@ -2348,51 +2349,67 @@ class ValidatingSchema(Schema):
 
 class UserContextSchema(Schema):
     is_owner = fields.Method("get_is_owner")
-    is_collab = fields.Function(lambda user, ctx: user in ctx["blog"])
+    is_collab = fields.Function(
+        lambda user: user in typing.cast(dict, CONTEXT.get())["blog"]
+    )
```

Comment:
For some reason I couldn't figure, mypy complains if I don't cast, but it does not complain about a later equivalent line where I didn't cast.

```py
likes_bikes = fields.Function(lambda obj: "bikes" in CONTEXT.get()["info"])
```